lib/employees/resolvers.ts

```diff
@@ -4,6 +4,19 @@ import { Resolver } from '@lib/resolvers'; // Resolver type
 import { EmployeeAlreadyExistsError } from '@lib/employees/errors'; // Employee errors
 import { DBErrorCode, getUniqueConstraintFailedFields } from '@lib/db/errors'; // Database errors
 
+/**
+ * Query for one employee in the internal-facing app given id
+ * @returns employee row
+ */
+export const employee: Resolver = async (_parent, _args, { prisma }) => {
```

Comment:
```suggestion
export const employee: Resolver = async (_parent, args, { prisma }) => {
```
Since we're using the `args` function parameter here, we don't want to prefix it with the underscore.